recipes/pyspf/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pyspf" %}
+{% set version = "2.0.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyspf-{{ version }}.tar.gz
+  sha256: 57a7ef01bda090173aafb6af0106251686ed73f03db4e911fcd34c57fc347186
+
+build:
```

Comment:
It's not noarch because of scripts, but the package is not being developed anymore, so it should only be built once.